src/frontend/src/components/EditDialog/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ConfirmDialog, {
+  ConfirmDialogProps,
+} from "src/components/ConfirmDialog";
+import { StyledSpan } from "src/components/DeleteDialog/style";
+
+interface Props extends Omit<ConfirmDialogProps, "onConfirm"> {
+  onEdit(): void;
+}
+
+const EditDialog = ({
+  onEdit,
+  open,
+  title,
+  ...props
+}: Props): JSX.Element | null => {
+  if (!open) return null;
+
+  const styledTitle = <StyledSpan>{title}</StyledSpan>;
+
+  return (
+    <ConfirmDialog
```

Comment:
Nit, continued: Along the same lines, maybe create a shared `<InteractableDialog>` component, then `ConfirmDialog`, `EditDialog`, and `DeleteDialog` are all basically just convenience components that handle that setup aspect. They feel really tightly bundled together right now, but the way there written implies they're less tightly coupled than they really are. You couldn't currently change `ConfirmDialog` without majorly disrupting the other two, but I don't think that's obvious at a glance currently.

Reply:
we may need to sync on monday about what your suggesting so we're on the same page, not quite sure i'm following

Reply:
i create a basic dialog that should hopefully be a better solution